common/api/ecschema-metadata.api.md

```diff
@@ -160,25 +160,26 @@ export interface CustomAttribute {
 export class CustomAttributeClass extends ECClass {
     constructor(schema: Schema, name: string, modifier?: ECClassModifier);
     // (undocumented)
-    get containerType(): CustomAttributeContainerType;
+    get appliesTo(): CustomAttributeContainerType;
     // (undocumented)
-    protected _containerType?: CustomAttributeContainerType;
+    protected _appliesTo?: CustomAttributeContainerType;
```

Comment:
I wasn't sure if this change was allowed but decided to go ahead because there should not be anyone deriving from this outside of the core libraries and the class is beta.

Reply:
@aruniverse

Reply:
since its beta go for it, just document it in nextver.md

Reply:
NextVersion.md updated